tests/jit/jit_models_test.py

```diff
@@ -90,26 +98,35 @@ def small_model_params():
     return params
 
 
+@pytest.mark.parametrize(
+    "model_def",
+    (
+        DCCRNet,
+        DCUNet,
+        DeMask,
+    ),
+)
 @pytest.mark.parametrize(
     "test_data",
     (
-        (torch.rand(240) - 0.5) * 2,
-        (torch.rand(1, 220) - 0.5) * 2,
-        (torch.rand(4, 256) - 0.5) * 2,
-        (torch.rand(1, 1, 301) - 0.5) * 2,
-        (torch.rand(3, 1, 290) - 0.5) * 2,
+        (torch.rand(4740) - 0.5) * 2,
+        (torch.rand(1, 4720) - 0.5) * 2,
+        (torch.rand(4, 4756) - 0.5) * 2,
+        (torch.rand(1, 1, 4801) - 0.5) * 2,
+        (torch.rand(3, 1, 4790) - 0.5) * 2,
```

Comment:
Can you explain the new shapes please? 
It's to give the name number of frames of STFT but have different wav size? Or what is the motivation?

Reply:
The problem is that DCUNet only works with certain shapes (T - 1 must be divisible by 16). I found these by trial and error. It's the same shapes as elsewhere + 4500.

Reply:
Ok, thanks. This is what I thought. 
Would it make sense to handle the "slicing" in the model? So that we don't have to be careful about it everywhere else? 

I'm worried that these tests pass because the effective input size, after the STFT, is the same. In this case, the scripting unit test is not useful.

Reply:
I'll have a deeper look at that asap

Reply:
This is what I had in mind. All the inputs have the same shape once out of the STFT, so this test ensure that if the input has the same shape, scripting keeps consistency. This is already good, but not enough. 
I'll keep digging